third_party/chromium/media/base/decoder_buffer.cc

```diff
@@ -181,6 +181,11 @@ scoped_refptr<DecoderBuffer> DecoderBuffer::CreateEOSBuffer() {
   return base::WrapRefCounted(new DecoderBuffer(NULL, 0, NULL, 0));
 }
 
+size_t DecoderBuffer::total_memory_usage() const {
+  return sizeof(DecoderBuffer) + side_data_size_ + size_ +
+         (decrypt_config_ ? decrypt_config_->total_memory_usage() : 0);
```

Comment:
nit: ... + data_size() + side_data_size() + ..., for two reasons:
1. Use accessors instead of the variables, to be in sync with other usages in the file.
2. To have them in the same order of definition (i.e. size() before side_data_size()).